### PR TITLE
Added Stack to Documentation Page

### PR DESCRIPTION
**Pull request by aleone2**
Added our current stack to the documentation page. Linked each of the technologies to their corresponding names as well as added a little bit about what each is doing and how each plays a role/has helped us in our developing. The descriptions blobs are pretty conversation I can change them to be more formal if we like. I also set up the tabs to be dynamic so we can add more to the page easily! Heres a preview. Can definetly go back and attempt to make prettier but let me know the thoughts on the layout.

image

### CODE REVIEW

**Comment by aleone2** [comment]
[Documentation Page (Add Stack)](https://trello.com/c/tMSmX9px/53-documentation-page-add-stack)

File: src/containers/Documentation.js [hunk 1]
```diff
@@ -1,26 +1,332 @@
...
+            Functionality like this is key in game situations where the host and players
+            need to be able to communicate back and forth seemingly instantly.
+          </p> 
+          <body>
+            <Link target="_blank" href="https://redux.js.org/" >
+              <u>Redux</u>
+            </Link>
+          </body>
+          <p align="center">
+          Redux is a state container for JS apps that allows us to maintain our mainy
```
Comment (jgordley) [inline src/containers/Documentation.js:256]: "mainy" -> "many"

File: src/containers/Documentation.js [hunk 2]
```diff
@@ -1,26 +1,332 @@
...
+      header: "Resources used in the Development of our FrontEnds!",
+      body:
+        <Box>
+          <body>
+            <Link target="_blank" href="https://mui.com/" >
+              <u>Material UI</u>
+            </Link>
+          </body>
+          <p>
+            Material UI has been an incredible source, that has helped us to make this website athetically pleasing (hopefully)
```
Comment (jgordley) [inline src/containers/Documentation.js:213]: "source" -> "resource"? potential typo

Comment (aleone2) [reply src/containers/Documentation.js:213]: yes indeed! nice catch.. fixing those typos right now

**Review by jgordley — APPROVED** [review]
Awesome work Abby, looks great!

**Review by chase-saca — APPROVED** [review]
The page looks great!

**Pull request merged**